lighthouse-core/report/html/report-styles.css

```diff
@@ -145,6 +147,30 @@
   --pwa-optimized-color-url: url('data:image/svg+xml;utf8,<svg xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><rect fill="%23FCE4EC" width="24" height="24" rx="12"/><path d="M5 5h14v14H5z"/><path fill="%23EC3F7A" d="M12 15.07l3.6 2.18-.95-4.1 3.18-2.76-4.2-.36L12 6.17l-1.64 3.86-4.2.36 3.2 2.76-.96 4.1z"/></g></svg>');
 }
 
+.lh-vars.dark {
+  --color-red-700: var(--color-red);
+  --color-green-700: var(--color-green);
+  --color-orange-700: var(--color-orange);
+
+  --color-black-200: var(--color-black-800);
+  --color-black-400: var(--color-black-600);
+  --color-black-600: var(--color-black-500);
+
+  --topbar-bg: var(--color-black);
+  --plugin-badge-bg: var(--color-black-800);
+  --header-bg-color: var(--color-black-900);
+  --env-item-bg: rgba(0, 0, 0, 0.5);
+
+  --body-background-color: var(--color-black-900);
+  --body-color: var(--color-black-100);
+
+  --highlighter-bg: var(--color-black-600);
+  --plugin-icon-url: var(--plugin-icon-url-dark);
+
+  --medium-50-gray: #757575;
```

Comment:
kinda floundering here

Reply:
this is the situation where naming based on usage instead of content would help make sense of things I think :)